src/Microsoft.AspNet.Mvc.ViewFeatures/Rendering/IHtmlHelper.cs

```diff
@@ -133,7 +135,7 @@ public interface IHtmlHelper
             string controllerName,
             object routeValues,
             FormMethod method,
-            bool suppressAntiforgery,
+            bool? suppressAntiforgery,
```

Comment:
Chatted in person with @ajaybhargavb and @dougbu . We want the new behavior, but we'll rename the parameter to match what the tag helper uses: `BeginForm(...., bool? antiForgery)`. And the behavior and docs will also be the same as the tag helper: true = on, false = off, null = smart.

We'll also post updates to the Announcements repo and related discussion item to let people know we renamed the property and flipped the behavior.